src/main/webapp/renderers/Step2Renderer.js

```diff
@@ -34,8 +34,8 @@ export default class Step2Renderer extends RendererInterface {
 
             this.parentElement.innerHTML += `
                 <div class="event-container">
-                    <h3>${event.name}</h3>
-                    <p>${event.startingTime} - ${event.endingTime}</p>
+                    <h3>${event.title}</h3>
+                    <p>${event.startTime} - ${event.endTime}</p>
```

Comment:
Just a suggestion:
When we switch to using Date for times, you can implement a function similar to formatDate on my PR (https://github.com/googleinterns/step220-2020/pull/21/files), so that it displays the date the way you prefer (ie. HH:MM AM/PM).

Reply:
Sure. I will keep that in mind